lib/opal/cli_runners/chrome.rb

```diff
@@ -85,15 +86,17 @@ def chrome_server_running?
       end
 
       def chrome_executable
-        ENV['GOOGLE_CHROME_BINARY'] || case RbConfig::CONFIG['host_os']
+        return ENV['GOOGLE_CHROME_BINARY'] if ENV['GOOGLE_CHROME_BINARY']
```

Comment:
It comes from the change in formatting. rubocop converted it to
```
ENV['GOOGLE_CHROME_BINARY'] || case RbConfig::CONFIG['host_os']	
                               when /mswin|msys|mingw|cygwin|bccwin|wince|emc/
                               # ...
```
But since we are going to revert this rule I'm going to revert this change as well.